spec/rubocop/result_cache_spec.rb

```diff
@@ -45,6 +45,28 @@ def abs(path)
       end
     end
 
+    context 'when offences are saved by autocorrect run' do
```

Comment:
Could you please include a link to the issue in a comment above this line, to indicate that this fixes a regression? 🙂

Reply:
And change spelling to "offenses".